### ISSUE TITLE
V2V should differentiate whether logs are moved or not generated

### ISSUE THREAD

**Issue by Yadnyawalkya** [ManageIQ/miq_v2v_ui_plugin#478]
**Description**
This is more of usability enhancement. V2V should differentiate whether logs are moved or not generated.
Sometime failed migration do not generate logs but while 'clicking on download logs' of details page error screen shows 'The location of transformation log was not set. Download of transformation log aborted'. This is somehow confusing since 'transformation log location is set'. We should show error with lines which differentiate whether logs are moved from transformation log directory or not generated yet.

**Version-Release number**
5.9.3.4

**Investigation**
If migration succeeds and log is created but moved, then getting following error.

`Failed to download "ytale-v2v-4-win10.log". Reason - "SCP did not finish successfully (1): scp: /var/log/vdsm/import/v2v-import-20180704T063253-126596.log: No such file or directory "`

and if migration fails and log is not get created, getting following error

```Failed to download "ytale-v2v-4-win10.log". Reason - "Conversion host was not found: ID []. Cannot queue the download of transformation log."```

Should be more specific as per description.

**Internal reference**
https://bugzilla.redhat.com/show_bug.cgi?id=1589265

**Comment by vconzola** [comment]
@AparnaKarve Can you take a look at this one when you get a chance? It's on Brett's list for 1.1 release.

**Comment by AparnaKarve** [comment]
@vconzola This particular error -
`Failed to download "ytale-v2v-4-win10.log". Reason - "Conversion host was not found: ID []. Cannot queue the download of transformation log."` 
occurs when we try to download the log too early in the migration process. 
So I am wondering that maybe we should keep the 'Download' button disabled until we have information about the log file location in the conversion host.

This error on the other hand -
`Failed to download "ytale-v2v-4-win10.log". Reason - "SCP did not finish successfully (1): scp: /var/log/vdsm/import/v2v-import-20180704T063253-126596.log: No such file or directory "`

is a proper download error, and the above message appears from the backend.
I think we should just have a more user friendly message here.

**Comment by vconzola** [comment]
@AparnaKarve I agree - disabling the button until the file is available is a good idea. In the case of playbook logs we should just not offer the migration log in the dropdown until it's available.

**Comment by vconzola** [comment]
@AparnaKarve Those messages look good to me.

**Comment by bthurber** [comment]
@vconzola @AparnaKarve checking status on dev complete for this

**Comment by AparnaKarve** [comment]
@bthurber @vconzola Created a PR https://github.com/ManageIQ/manageiq-v2v/pull/793 for this which would be reviewed soon. Thanks!

**Comment by bthurber** [comment]
@AparnaKarve thanks!  Looks like the PR is still open.  Can I assist with locating a reviewer?

@vconzola ping for visibility.
